en/config/safety.md

```diff
@@ -0,0 +1,227 @@
+
+The relevant parameters for all vehicles shown below (also see [GPS Failure navigation parameters](../advanced_config/parameter_reference.md#gps-failure-navigation)):
+
+Parameter | Description
+--- | ---
+[COM_POS_FS_DELAY](../advanced_config/parameter_reference.md#COM_POS_FS_DELAY) | Delay after loss of position before the failsafe is triggered.
+[CBRK_GPSFAIL](../advanced_config/parameter_reference.md#CBRK_GPSFAIL) | Circuit breaker that can be used to disable GPS failure detection.
+[NAV_GPSF_LT](../advanced_config/parameter_reference.md#NAV_GPSF_LT) | Loiter time (waiting for GPS recovery before it goes into flight termination). Set to 0 to disable.
+
+Parameters that only affect Fixed Wing vehicles:
+
+Parameter | Description
+--- | ---
+[NAV_GPSF_P](../advanced_config/parameter_reference.md#NAV_GPSF_P) | Fixed pitch angle while circling.
+[NAV_GPSF_R](../advanced_config/parameter_reference.md#NAV_GPSF_R) | Fixed roll/bank angle while circling.
+[NAV_GPSF_TR](../advanced_config/parameter_reference.md#NAV_GPSF_TR) | Thrust while circling.
+
+
+### Offboard Loss Failsafe
+
+The *Offboard Loss Failsafe* is triggered if the offboard link is lost while under Offboard control. 
+Different failsafe behaviour can be specified based on whether or not there is also an RC connection available.
+
+The relevant parameters are shown below:
+
+Parameter | Description
+--- | ---
+[COM_OF_LOSS_T](../advanced_config/parameter_reference.md#COM_OF_LOSS_T) | Delay after loss of offboard connection before the failsafe is triggered.
+[COM_OBL_ACT](../advanced_config/parameter_reference.md#COM_OBL_ACT) | Failsafe action if no RC is available: Land, Loiter, Return
+[COM_OBL_RC_ACT](../advanced_config/parameter_reference.md#COM_OBL_RC_ACT) | Failsafe action if RC is available: Position mode, Altitude mode, Manual, Return, Land, Loiter.
+
+
+### Mission Failsafe
+
+The Mission Failsafe checks prevent a previous mission being started at a new takeoff location or if it is too big (distance between waypoints is too great).
+The failsafe action is that the mission will not be run.
+
+The relevant parameters are shown below:
+
+Parameter | Description
+--- | ---
+* [MIS_DIST_1WP](../advanced_config/parameter_reference.md#MIS_DIST_1WP) | The mission will not be started if the current waypoint is more distant than this value from the home position. Disabled if value is 0 or less.
+* [MIS_DIST_WPS](../advanced_config/parameter_reference.md#MIS_DIST_WPS) | The mission will not be started if any distance between two subsequent waypoints is greater than this value.
+
+
+### Traffic Avoidance Failsafe
+
+The Traffic Avoidance Failsafe allows PX4 to respond to transponder data (e.g. from ADSB transponders).
+
+The relevant parameters are shown below:
+
+Parameter | Description
+--- | ---
+[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Set the failsafe action: Disabled, Warn, Return, Land.
+
+
+### Adaptive QuadChute Failsafe
```

Comment:
Is this worth including? 

I think that it is VTOL only, and that it is a failsafe for when the altitude estimate diverges too far from the setpoint - ie vehicle can no longer achieve desired height in FW mode. Correct?

Reply:
Yes it is important. But I'm not sure in which cases people use it.

Reply:
OK. Who would know?